pkg/sdk/testint/resource_monitors_integration_test.go

```diff
@@ -270,30 +319,23 @@ func TestInt_ResourceMonitorAlter(t *testing.T) {
 		newTriggers = append(newTriggers, sdk.TriggerDefinition{Threshold: 30, TriggerAction: sdk.TriggerActionNotify})
 
 		creditQuota := 100
-		alterOptions := &sdk.AlterResourceMonitorOptions{
+		err := client.ResourceMonitors.Alter(ctx, resourceMonitor.ID(), &sdk.AlterResourceMonitorOptions{
 			Set: &sdk.ResourceMonitorSet{
 				CreditQuota: &creditQuota,
 				NotifyUsers: &sdk.NotifyUsers{
-					Users: []sdk.NotifiedUser{{Name: "ARTUR_SAWICKI"}},
+					Users: []sdk.NotifiedUser{{Name: sdk.NewAccountObjectIdentifier("JAN_CIESLAK")}},
```

Comment:
xD

Reply:
hihi

Reply:
Yes, that's the reason my name was there in the first place :D (check https://github.com/Snowflake-Labs/terraform-provider-snowflake/blob/78ecb62935c1a40611418e59ed5f0d644cf85f03/pkg/resources/email_notification_integration_acceptance_test.go#L14)